src/charm.py

```diff
@@ -6,15 +6,15 @@
 
 """Prometheus Scrape Configuration charm.
 
-The Prometheus Scrape Configuration is a charm that
-allows to override configurations for the scrape jobs
-generated by charms that provide relations using the
-'metrics-endpoint' relation.
+The Prometheus Scrape Configuration charm
+allows overriding configuration of scrape jobs
+from any metrics provider the provides these metrics
+through the 'metrics-endpoint' relation using the
+`prometheus_scrape` interface.
```

Comment:
```suggestion
through the 'metrics-endpoint' relation.
```

Reply:
I do want to make clear what the interface is, since that is more important.